shop-local/src/components/Store-info.js

```diff
@@ -1,17 +1,28 @@
 import "../App.css"
 
-const StoreInfo = ({ name, address, description, openTime, closeTime, phoneNumber, instagram, facebook }) => {
+
+// const StoreInfo = ({ name, address, description, openTime, closeTime, phoneNumber, instagram, facebook }) => {
```

Comment:
Seems like you don't need this comment block then? Would be best to take it out if not needed

Reply:
This where the page for one store is created but it does now handle the json object correctly yet. Is it okay to leave it there until we get the data retrieval?

Reply:
Sounds good